docs/userGuide/syntax/links.mbdf

```diff
@@ -109,11 +109,27 @@ Within `index.md`, we can also display the image using
 
 <div class="indented">
 
-Links will be validated when generating a site and a warning will be displayed in the console for every link that is invalid.
+Links will be validated when generating a site and a warning will be displayed in the console for every link that is invalid. 
+
 
 <box type="warning">
 
-If you wish to disable validation for a specific link, you may use the `no-validation` attribute.
+**Disabling link validation**
+
+Link validation is enabled by default. 
+
+If you wish to disable this feature **entirely**, you may simply modify your `site.json` as follows. 
+
+{{ icon_example }}
+```js
+...
+"intrasiteLinkValidation": {
+	"enabled": false
+},
+...
+```
+
```

Comment:
MarkBind is supposed to be good at reuse. Try to avoid duplicating the similar content by using an available reuse mechanism e.g., an `include`.

Reply:
Noted, prof! Can I clarify which duplicated part are u referring to here?

Reply:
Although not exactly the same text, this description of how to disable it is also in site.json, right? Consider tweaking the text to fit both places and reusing that in both places.

Reply:
Understood! Will look into that 👍